src/main/java/impl/TransactionServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package impl;
+
+import java.util.List;
+import model.FruitTransaction;
+import service.OperationStrategy;
+import service.ParseService;
+import service.ReadService;
+import service.ReportService;
+import service.WriteService;
+import strategy.OperationHandler;
+
+public class TransactionServiceImpl {
```

Comment:
```suggestion
public class TransactionServiceImpl implements TransactionService {
```

Reply:
but I don't have interface TransactionService

Reply:
Okay, I will create this interface and method processTransaction and after that, override method in class TransactionServiceImpl